src/coordinatorapi.cpp

```diff
@@ -755,11 +755,12 @@ kvdb::KVDBResponse
 kvdbRequest(DmtcpMessage const& msg,
             string const& key,
             string const& val,
-            string *oldVal)
+            string *oldVal,
+            bool useNsSock)
```

Comment:
Can we have a more meaningful name than `useNsSock`?  :-)
Longer variable names are good if they help people like me to read the code.  "Sock" probably means "Socket", and the extra two characters are a small cost for readability.
    I guess you were already using `nsSock`, but I would have to study the code further to figure out what an `nsSock` is.  Can we also change that name to something more meaningful?